assets/activation-scripts/activate

```diff
@@ -39,7 +39,7 @@ wait_for_services_socket () {
   local status
   status=$(poll_services_status "$socket_file")
   while [ "$status" == "$NOT_READY" ]; do
-    sleep 0.1
+    sleep 0.02
```

Comment:
Could you describe this change?

Reply:
The use of `sleep` always brings with it a code smell; could we use `mkfifo` as a guard here?